components/helm-broker/cmd/controller/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/kyma-project/kyma/components/helm-broker/internal/controller"
+	"github.com/kyma-project/kyma/components/helm-broker/internal/storage"
+	"github.com/kyma-project/kyma/components/helm-broker/pkg/apis"
+
+	envs "github.com/kyma-project/kyma/components/helm-broker/internal/config"
+	"github.com/kyma-project/kyma/components/helm-broker/internal/controller/broker"
+	"github.com/kyma-project/kyma/components/helm-broker/platform/logger"
+	"github.com/sirupsen/logrus"
```

Comment:
`logrus` is used only inside `fatalOnError` function, can we reduce the log library?

Reply:
we use logrus underneath in our logger so it won't make a difference.